schemas/README.md

```diff
@@ -1,17 +1,146 @@
 
-To include a new schema plugin in a build, copy the schema folder
-here and add it to the copy-schemas execution in web/pom.xml.
+3. Add a profile to `web/pom.xml` unpacking your schema plugin.
+   
+   ```xml
+    <profile>
+      <id>hnap</id>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-dependency-plugin</artifactId>
+            <dependencies>
+              <dependency>
+                <groupId>${project.groupId}</groupId>
+                <artifactId>schema-iso19139.ca.HNAP</artifactId>
+                <version>${gn.schemas.version>}</version>
+              </dependency>
+            </dependencies>
+            <executions>
+              <execution>
+                <id>iso19139.ca.HNAP-resources</id>
+                <phase>process-resources</phase>
+                <goals><goal>unpack</goal></goals>
+                <configuration>
+                  <encoding>UTF-8</encoding>
+                  <artifactItems>
+                    <artifactItem>
+                      <groupId>org.geonetwork-opensource</groupId>
+                      <artifactId>schema-iso19139.ca.HNAP</artifactId>
+                      <version>${gn.schemas.version>}</version>
+                      <type>jar</type>
+                      <overWrite>false</overWrite>
+                      <outputDirectory>${basedir}/src/main/webapp/WEB-INF/data/config/schema_plugins</outputDirectory>
+                      <includes>src/main/plugin</includes>
+                    </artifactItem>
+                  </artifactItems>
+                </configuration>
+              </execution>
+            </executions>
+          </plugin>
+        </plugins>
+      </build>
+    </profile>
+   ```
 
+4. Use your profile, for example `mvn process-resources -Phnap`, to extract plugin from artifact during build.
+
+## Option: Fork and add a plugin to the build
+
+To include a new schema plugin in a build:
+
+1. Add schema plugin:
+   
+   * As a git submodule if you are making a fork
+   * As a folder, using `.gitignore` to avoid accidentally commiting.
+`
+2. Use a profile (activated by your folder being present) to include your schema plugin to the list of modules in `pom.xml`.
+   
+   ```xml
+   <profiles>
+     <profile>
+       <id>hnap</id>
+       <activation>
+       <file>
+         <exists>iso19139.ca.HNAP</exists>
+       </file>
+       </activation>
+       <modules>
+         <module>iso19139.ca.HNAP</module>
+       </modules>
+     </profile>
+   </profiles>
+   ```
+
+3. Add your schema to the `copy-schemas` execution in `web/pom.xml`.
+   
+   ```xml
+   <resource>
+     <directory>${project.basedir}/../schemas/iso19139.ca.HNAP/src/main/plugin</directory>
+     <targetPath>${basedir}/src/main/webapp/WEB-INF/data/config/schema_plugins
+     </targetPath>
+   </resource>
+   ```
+
+4. Use `mvn process-resoruces -Dcopy` to activate `copy-schemas` execution.
```

Comment:
Small typo in `resoruces`

Reply:
Thanks, especially for README.md please use your access as a committer to modify this PR directly (you can click edit when reviewing).